### ISSUE TITLE
Updating environment

### ISSUE THREAD

**Issue by zhenghanQ** [gablab/analysis#4]
Hi, 

I just edited my existing environment file in order to update the freesurfer version. But when I push the changes to masters: git push origin masters, I saw:
error: src refspec masters does not match any.
error: failed to push some refs to 'git@github.com:zhenghanQ/mindhive.git'

Has anyone seen this before?

Thanks!

Zhenghan

**Comment by doreenr** [comment]
Did you commit your changes first?

http://byte.kde.org/~zrusin/git/git-cheat-sheet-large.png

**Comment by zhenghanQ** [comment]
Thanks for the helpful link Doreen~! 

Yes, I did commit my changes.
As a matter of fact, I committed twice. The second time it says the following, but now "git push origin master" works!

On branch master
Your branch is ahead of 'origin/master' by 1 commit.
Changes not staged for commit:
(use "git add <file>..." to update what will be committed)
(use "git checkout -- <file>..." to discard changes in working directory)
modified:   .env/.BILD_environment
 Untracked files:
   (use "git add <file>..." to include in what will be committed)

.env/.BILD_environment~
.env/.CASL_environment~
.projects~
no changes added to commit (use "git add" and/or "git commit -a")
